DatabaseMySQL/dumps/Dump20221016.sql

```diff
@@ -0,0 +1,290 @@
+  PRIMARY KEY (`EXAM_ID`),
+  KEY `SUBJECTID_FK_idx` (`SUBJECT_ID`),
+  CONSTRAINT `SUBJECTID_FK` FOREIGN KEY (`SUBJECT_ID`) REFERENCES `subjects` (`SUBJECT_ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb3;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `exams`
+--
+
+LOCK TABLES `exams` WRITE;
+/*!40000 ALTER TABLE `exams` DISABLE KEYS */;
+INSERT INTO `exams` VALUES (1,1,'desciption',10,100,10);
+/*!40000 ALTER TABLE `exams` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `questions`
+--
+
+DROP TABLE IF EXISTS `questions`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `questions` (
+  `QUESTION_ID` bigint NOT NULL AUTO_INCREMENT,
+  `SUBJECT_ID` bigint NOT NULL,
+  `LEVEL` int NOT NULL DEFAULT '1',
+  `CONTENT` text,
+  PRIMARY KEY (`QUESTION_ID`),
+  KEY `SUBJECTID_FK_idx` (`SUBJECT_ID`),
+  CONSTRAINT `SUBJECTID_FK1` FOREIGN KEY (`SUBJECT_ID`) REFERENCES `subjects` (`SUBJECT_ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb3;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `questions`
+--
+
+LOCK TABLES `questions` WRITE;
+/*!40000 ALTER TABLE `questions` DISABLE KEYS */;
+INSERT INTO `questions` VALUES (1,1,1,'Vong 1');
+/*!40000 ALTER TABLE `questions` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `questiopn_answers`
+--
+
+DROP TABLE IF EXISTS `questiopn_answers`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `questiopn_answers` (
+  `QUESTIOPN_ANSWER_ID` bigint NOT NULL AUTO_INCREMENT,
+  `QUESTION_ID` bigint NOT NULL,
+  `SUBJECT_ID` bigint NOT NULL,
+  `IS_CORRECT` bit(1) NOT NULL DEFAULT b'0',
+  `CONTENT` text,
+  PRIMARY KEY (`QUESTIOPN_ANSWER_ID`),
+  KEY `QUESTIONID_FK_idx` (`QUESTION_ID`),
+  KEY `SUBJECTID_FK_idx` (`SUBJECT_ID`),
+  CONSTRAINT `QUESTIONID_FK` FOREIGN KEY (`QUESTION_ID`) REFERENCES `questions` (`QUESTION_ID`),
+  CONSTRAINT `SUBJECTID_FK2` FOREIGN KEY (`SUBJECT_ID`) REFERENCES `subjects` (`SUBJECT_ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8mb3;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `questiopn_answers`
+--
+
+LOCK TABLES `questiopn_answers` WRITE;
+/*!40000 ALTER TABLE `questiopn_answers` DISABLE KEYS */;
+INSERT INTO `questiopn_answers` VALUES (1,1,1,_binary '','Do vui');
+/*!40000 ALTER TABLE `questiopn_answers` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `rooms`
+--
+
+DROP TABLE IF EXISTS `rooms`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `rooms` (
+  `ROOM_ID` bigint NOT NULL AUTO_INCREMENT,
+  `EXAM_ID` bigint NOT NULL,
+  `PASSWORD` varchar(50) NOT NULL,
+  `TITLE` text,

```

Comment:
TITLE | VARCHAR(50) NOT NULL

Thiếu CONTENT | TEXT NULL DEFAULT NULL | Thông tin về phòng thi